docs/_interface/Bash-Autocomplete.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Bash Autocomplete
+sort_key: C
```

Comment:
This should fix the issue you described (default page for `Using geth`)

```suggestion
sort_key: B
```

Reply:
It just ordered the `Bash Autocompletion`'s position 😢  Unfortunately it doesn't solve the issue... What we want is that:

When we click in the option `Using Geth` redirects to `Command-line Options` instead of `Bash Autocompletion`.

Thanks for your suggestion @s1na , I will put the print here just to show you what happened:

![image](https://user-images.githubusercontent.com/39157101/163708173-663bd606-d4a1-474c-b193-8df02157ca9c.png)